OracleGoldenGate/21c/Dockerfile

```diff
@@ -14,7 +14,8 @@ COPY          install-*.sh      /tmp/
 COPY          ${INSTALLER}      /tmp/installer.zip
 COPY          bin/              /usr/local/bin/
 
-RUN           bash -c  /tmp/install-prerequisites.sh && \
+RUN           chmod 755 /usr/local/bin/*             && \
```

Comment:
This should be handled by the file mode stored in Git. Did you run into problems on Windows maybe?

Reply:
No, this was built on a linux machine.

Reply:
OK. Well, the `install-prerequisites.sh` script is required. I believe the changes to `Dockerfile` should be reverted,

Reply:
The use of `chmod` is not required. All shell scripts and other executables already have the executable bit set in this Git repository. These changes can be reverted.

Reply:
I took the chmod out and if needed can be done via Dockerfile add-on or user script